grafana/dashboard/user/storage.py

```diff
@@ -34,12 +34,12 @@ def _(outer_panels: Panels):
                     ],
                 ),
                 panels.timeseries_kilobytes(
-                    "Materialized View Size",
+                    "Estimated Materialized View Size (before compression)",
                     "The storage size of each materialized view",
```

Comment:
Would it be better if we clarify that this metric includes mv and all associated state tables?

Reply:
Good point. I modified the description to reflect that.